src/opentime/version.h.in

```diff
@@ -3,7 +3,7 @@
 
 #pragma once
 
-#define OPENTIME_VERSION v1_0
+#define OPENTIME_VERSION v@PROJECT_VERSION_MAJOR@_@OpenTimelineIO_VERSION_MINOR@
```

Comment:
I don't think we need to follow the `v1_0`, I think your original changes were good; maybe something like:
```
#define OPENTIME_VERSION_MAJOR @OTIO_VERSION_MAJOR@
#define OPENTIME_VERSION_MINOR @OTIO_VERSION_MINOR@
#define OPENTIME_VERSION_PATCH @OTIO_VERSION_PATCH@
#define OPENTIME_VERSION "@OTIO_VERSION@"
```